$NetBSD$

don't fail if xattrs are not supported by the filesystem

--- test/bbackupd/testbbackupd.cpp.orig	2011-01-08 21:35:24.000000000 +0000
+++ test/bbackupd/testbbackupd.cpp
@@ -310,9 +310,9 @@ bool attrmatch(const char *f1, const cha
 		if(!readxattr_into_map(f1, xattr1)
 			|| !readxattr_into_map(f2, xattr2))
 		{
-			return false;
+			/*return false*/;
 		}
-		if(!(xattr1 == xattr2))
+		else if(!(xattr1 == xattr2))
 		{
 			return false;
 		}
